_config.yml

```diff
@@ -69,3 +61,7 @@ teammembers:
     role: Computer Science
     role2: (undergraduate student)
     image: sinclair_hudson.jpg
+  - name: Maria Mihu
```

Comment:
I believe we have an alphabetical ordering in each group, can we do that here?

Reply:
sure thing!